extensions/adobe/experience/decisioning/option-propositions-total.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/option-propositions-total",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Option propositions total",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/metricdefinition"],
+  "description": "The total number a particular option was proposed so far across all profiles.",
```

Comment:
Not parsing this description, can you have a look at it please.

Reply:
The total number a particular option was proposed (so far) across all profiles. The option is the entity to which the metrics is attached.